Documentation/Books/Manual/Programs/Arangodump/Maskings.md

```diff
@@ -88,14 +109,140 @@ or
 
     "path": "`name.with.dots`"
 
-xify_front
-----------
+If the attribute value is an array the masking is applied to all the
+array elements.
+
+### Example
```

Comment:
Headlines must be unique per document. The easiest way to fix is to use bold text instead:

```suggestion
**Example**
```

Reply:
done